packages/hoppscotch-backend/src/admin/admin.resolver.ts

```diff
@@ -281,10 +103,30 @@ export class AdminResolver {
     })
     userUID: string,
   ): Promise<boolean> {
-    const invitedUser = await this.adminService.removeUserAccount(userUID);
-    if (E.isLeft(invitedUser)) throwErr(invitedUser.left);
-    return invitedUser.right;
+    const removedUser = await this.adminService.removeUserAccount(userUID);
+    if (E.isLeft(removedUser)) throwErr(removedUser.left);
+    return removedUser.right;
   }
+
```

Comment:
I dont like the fact we have 2 seperate mutations to do the same operation, I understand maybe its done this way due to backwards compatability reasons but this mutation is only used on one location in the admin app so I think changing it is there is fine.